core/services/workflows/syncer/workflow_registry.go

```diff
@@ -269,7 +269,7 @@ func (w *workflowRegistry) readRegistryEvents(ctx context.Context, reader Contra
 			KeyFilter: query.KeyFilter{
 				Key: string(et),
 				Expressions: []query.Expression{
-					query.Confidence(primitives.Finalized),
+					query.Confidence(primitives.Unconfirmed),
```

Comment:
@justinkaseman Sorry... two-liner apparently -- you need to modify the call to the chain reader in loadWorkflows too